Software/real_time/ROS_RoboBuggy/src/robobuggy/msg/Brake.msg

```diff
@@ -1,5 +0,0 @@
-Header header
-
-bool brake_state
-bool brake_cmd_teleop
```

Comment:
I'd like to leave this in, it gives the electrical team insight into whether the radio controller is working properly

Reply:
Maybe name it something like `rc_brake_cmd_status`

Reply:
I'm putting teleop_brake_cmd_status and rc_brake_cmd_status in the Diagnostics message, since they're not relevant to either feedback or control. brake state and brake command are the ones relevant to Feedback/Command.